### DISCUSSION TITLE
Read another package.json

### DISCUSSION THREAD

**Discussion by skernke** [renovatebot/renovate#20254] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

GitHub Enterprise Server

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

We have a base custom npm package that provides methods for our backend nestjs services.
Our problem right now is if you update the base repository on Friday. But first, upgrade the other repository which uses the other repository on Monday it could happen that in between there was a new version release for some nestjs packages.
So we get conflicts since the version could/is newer than the one in the base repository.
is it possible somehow to read another package.json and stick to that version if the same packages are installed?
so something like sync package.json

### What you tried so far.

I only read the documentation but didn't find anything if it is already possible.

**Comment by RahulGautamSingh** [comment]
`But first, upgrade the other repository which uses the other repository on Monday` 
Hard to understand this. Could you please explain this using an example with names?

**Comment by skernke** [comment]
For example
we have our own package
myPackage v1.0.0

which contains "@nestjs/common": "9.3.0"
than renovate suggested to upgrade to version **9.3.1**
so we update it an release v1.0.1 of our own package.

now as we update another project from us which includes myPackage from version v1.0.0 to version v1.0.1
there was in between another version release by nestjs so at the same time it suggests to upgrade to "@nestjs/common": "**9.3.2**"
and than we would a version difference between our nestjs version in our myPackage which would include  "@nestjs/common": "9.3.1" and it would try upgrade the others project nestjs version "@nestjs/common": "9.3.2".
I hope I'm not confusing you now even more.

so somehow it would be nice if I could configure that renovate doesn't update the @nestjs/common in the second project only to the same version like in the myPackage project
